build.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
-git submodule update --init --recursive
```

Comment:
After removing this line during the first build protobuf submodule is not installing and the build process is failing. Here is the error I got during the first build:
```macros@macros:~/Job/PR_checks/hardware-wallet$ ./build.sh
Using default tag: latest
latest: Pulling from skycoinproject/skywallet\
Digest: sha256:f14b4f7958988880a8bf0b9d28430d7069d32060dd12aac62ca64319e1de32a8
Status: Image is up to date for skycoinproject/skywallet:latest
docker.io/skycoinproject/skywallet:latest
make -C tiny-firmware/protob/ build-c
make[1]: Entering directory '/hardware-wallet/tiny-firmware/protob'
make[1]: *** No rule to make target 'build-c'.  Stop.
make[1]: Leaving directory '/hardware-wallet/tiny-firmware/protob'
Makefile:92: recipe for target 'build-deps' failed
make: *** [build-deps] Error 2
```

Running `git submodule update --init --recursive` command before `./build.sh` fixed the problem